oaknational/Oak-Web-Application
# Fix/lesq 679/programme listing [LESQ-679]

## Description

Music year: 2001

- update programme listing query to accept isLegacy param
- remove hacky method previously used to keep legacy and new programmes distinct

## Issue(s)

Fixes KS4 Biology programmes not appearing

## How to test

1. Go to https://deploy-preview-2341--oak-web-application.netlify.thenational.academy/teachers/key-stages/ks4/subjects/biology/programmes
3. You should see tiers and examboards
4. also check other programmes which have different tiers/examboards between legacy and new programmes
